qutebrowser/browser/history.py

```diff
@@ -80,6 +80,7 @@ class CompletionMetaInfo(sql.SqlTable):
 
     KEYS = {
         'force_rebuild': False,
```

Comment:
Like you mentioned in your comment, I think you can get rid of `force_rebuild` and the old `_on_config_changed` entirely.